.github/workflows/cmake.yml

```diff
@@ -16,6 +16,8 @@ jobs:
             cmake_options:
               -DPA_USE_OSS=ON
               -DCMAKE_TOOLCHAIN_FILE=vcpkg/scripts/buildsystems/vcpkg.cmake
+              -DOSS=ON
```

Comment:
What is the purpose of this `-DOSS=ON`? Is this needed. I'm not sure whether this is an unrelated change.

Reply:
Was an incorrect rebase on my side